CONTRIBUTING.md

```diff
@@ -30,3 +36,133 @@ to review and merge.
+OpenOakland meetings. Our best work gets done when folks are available, not at
+the meetings.
+
+
+### Kanban board
+
+Because we don't really time-box our work in Sprints, we lean more towards
+[Kanban](https://www.atlassian.com/agile/kanban) agile methodology. Work is
+described as stories and are put into the Backlog. The Backlog is groomed and
+prioritized into the Ready queue. Team members pull work from Ready and only
+work on one or two things at a time (work-in-progress limits).
+
+We use GitHub milestones to focus a set of work for feature releases and use
+[semantic versioning](https://semver.org) when we release.
+
+
+### Story lifecycle
+
+**Stories** represent tactical increments of individually-valuable work
+deliverable by the team within a single iteration... often an isolated change in
+functionality aimed at achieving a goal for a particular kind of stakeholder,
+whether customer, user, or operator/admin. Stories are tracked on the [Kanban
+Board](https://github.com/openoakland/dos-census-events/projects/1)
+and progress through these columns.
+
+- New
+- Backlog
+- Ready
+- In progress
+- Done
+
+
+### Definition of "Done"
+
+An agile "Definition of Done" (DoD) captures the team's agreed-upon standards
+for how we get work done at a consistent level of quality. Having a DoD ensures
+that non-functional requirements (NFRs) don't have to be re-litigated for every
+piece of work taken on, cards can be focused on just the relevant details, and
+new team members aren't surprised by assumed expectations of their colleagues.
+
+
+#### Column exit criteria
+
+Our DoD is broken up into a set of statements that should be true
+for each card before it moves to the next column on the board.
+
+Before advancing a card from one column to the next on the board, it should meet
+the "exit criteria" for the current column, which are listed below.
+
+
+### Columns
+
+
+#### New
+
+New issues that need to be triaged. If the issue is low priority, we remove it
+from the Kanban board (don't worry, it's not lost). Otherwise, it is moved to
+the Backlog to be groomed.
+
+
+##### Exit criteria
+
+- Relevant points from any discussion in the comments is captured in the initial
+  post.
+- Decision is made to move to the Backlog, remove it from the board, or close it.
+
+
+#### Backlog
+
+Work that we are planning on doing and will groom and schedule.
+
+
+##### Exit criteria
+
+- Indicate the intended benefit and who the story is for in the "as a ..., I want
+  ..., so that ..." form.
+- Acceptance criteria is defined.
+
+
+#### Ready
+
+Prioritized stories ready for a team member to begin work. Stories in this
+column should be well-defined and ready to begin work.
+
+
+##### Exit criteria
+
+- No info or assistance is needed from outside the team to start work and likely
+  finish it.
+- There's capacity available to work on the story (e.g., this column is a buffer
+  of shovel-ready work).
+
+
```

Comment:
Don’t think we need to have limit.

Reply:
We can remove the limit, but I find that it's easy for volunteers and new Brigade members to take on too many issues and then work stalls. Work-in-progress limits help to make sure people finish what they start.